### PR TITLE
Various worker cleanups.

### PR DESCRIPTION
**Pull request by robertwb**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by robertwb** [comment]
R: @tvalentyn

**Comment by tvalentyn** [comment]
Looks like some precommit tests are failing.

**Comment by robertwb** [comment]
Fixed. PTAL.

File: sdks/python/apache_beam/runners/portability/fn_api_runner.py [hunk 1]
```diff
@@ -134,22 +110,32 @@ def __init__(self, pre_grouped_coder, post_grouped_coder, windowing):
...
+      key, value = windowed_key_value.value
+      self._table[key_coder_impl.encode(key)].append(
+          value if is_trivial_windowing
+          else windowed_key_value.with_value(value))
 
   def __iter__(self):
     output_stream = create_OutputStream()
-    trigger_driver = trigger.create_trigger_driver(self._windowing, True)
+    if self._windowing.is_default():
+      globally_window = GlobalWindows.windowed_value(None).with_value
```
Comment (tvalentyn) [inline sdks/python/apache_beam/runners/portability/fn_api_runner.py]: Did you mean to call this global_window?

Comment (robertwb) [reply sdks/python/apache_beam/runners/portability/fn_api_runner.py]: This is a verb, it's the function that takes a value and globally windows it.

**Pull request merged**